week-1/explore/objects.md

```diff
@@ -4,13 +4,15 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
-What output do you get?
-
-Now enter just `console` in the Console, what output do you get back?
-
+What output do you get? 
+// ƒ log() { [native code] }
+Now enter just `console` in the Console, what output do you get back? 
+// list of functions and methods
 Try also entering `typeof console`
 
 Answer the following questions:
 
 What does `console` store?
+// console is not for storing, the console is for outputting and displaying. 
 What does the syntax `console.log` or `console.assert` mean? In particular, what does the `.` mean?
+// the console object is calling the functions log and assert. // The "." is to call a function.
```

Comment:
This is a really good explanation. I did not know this. I think the whole group can benefit from this explanation.